src/components/Main.js

```diff
@@ -29,32 +34,70 @@ export default function Main() {
   const [coinList, setCoinList] = useState([]);
   const [searchCoin, setSearchCoin] = useState("");
 
+  const dispatch = useDispatch();
+
+  const tickerCoinList = useSelector((state) => state.socket.coinList);
+  const realTimeCoin = useSelector((state) => state.socket.socketCoin);
+
+  useEffect(() => {
+    const myCoin = "ALL";
+
+    const fetchCoinList = function () {
+      dispatch(requestCoinList(myCoin));
+    };
+
+    fetchCoinList();
```

Comment:
```suggestion
    dispatch(requestCoinList(myCoin));
```
fetchCoinList 라는 함수를 선언하지 않고 그냥 dispatch 만 바로 보낼 수 있을 거 같습니다 !